### PR TITLE
Add conditionals to form via JS Driver

### PR DESCRIPTION
**Pull request by Jade-GG**
Also adds a simple `select` field

### CODE REVIEW

File: src/Forms/Vue.php
Comment (royduin) [inline src/Forms/Vue.php:1]: Maybe move it to another subdirectory just like Statamic: `Rapidez\Statamic\Forms\JsDrivers` so we know what it is.

Comment (Jade-GG) [reply src/Forms/Vue.php:1]: Ah you're right, I'll stick to the consistency with the Statamic version (This was the path I stole from the Vue example)

File: resources/views/form_fields/select.blade.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<x-input.select
```
Comment (royduin) [inline resources/views/form_fields/select.blade.php:1]: This component doesn't exist, see: https://github.com/rapidez/core/tree/master/resources/views/components

File: resources/js/package.js
Comment (royduin) [inline resources/js/package.js:1]: How big is this? Maybe move it to a Vue component file which will be lazy loaded? This way it's bundled and available on all pages.

File: resources/views/form_fields/select.blade.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    :class="$field['error'] ? 'border-red-500' : ''"
+    :required="in_array('required', $field['validate'] ?? [])"
+    v-model="formData.{{ $field['handle'] }}"
+>
+    @if (!($field['default'] ?? ''))
+        <option
+            value="null"
+            disabled
+            selected
+        >{{ $field['placeholder'] ?? '' }}</option>
```
Comment (royduin) [inline resources/views/form_fields/select.blade.php:14]:
```suggestion
        <option value="null" disabled selected>
            {{ $field['placeholder'] ?? '' }}
        </option>
```

File: resources/views/form_fields/select.blade.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    @endif
+    @foreach ($field['options'] ?? [] as $option => $label)
+        <option
+            value="{{ $option }}"
+            {{ $option == ($field['default'] ?? '') ? 'selected' : '' }}
+        >{{ $label }}</option>
```
Comment (royduin) [inline resources/views/form_fields/select.blade.php:20]:
```suggestion
        <option value="{{ $option }}" @selected($option == $field['default'] ?? '')>
            {{ $label }}
        </option>
```

File: resources/js/package.js
Comment (Jade-GG) [reply resources/js/package.js:1]: It's a very small component, but I guess these small components do add up over time so it's worth to lazy load it (especially given it's only used on very specific pages)

File: resources/js/package.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import '/public/vendor/statamic/frontend/js/helpers.js'
```
Comment (royduin) [inline resources/js/package.js:1]: This is still within the base bundle, can we move this / everything to the component?

File: resources/js/package.js
Comment (royduin) [reply resources/js/package.js:1]: More for the included Statamic JS, not sure how big that is?

File: resources/js/package.js (hunk 4 above)
Comment (Jade-GG) [reply resources/js/package.js:1]: The helpers import is required for the `window.Statamic` function I'll try to mess with it to see if I can minimize it though

Comment (royduin) [reply resources/js/package.js:1]: Can't we just move everything to the component?

**Pull request merged**